### PR TITLE
Added: Xml Response + Fixed: links

### PR DESCRIPTION
**Pull request by JanGalek**
- Added: XmlResponse 
- Fixed: links

### CODE REVIEW

File: src/Response/XmlResponse.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+use Nette;
+use Nette\Application\Responses\TextResponse;
+
+/**
+ * @author Jan Galek <admin@gcore.cz>
+ */
+class XmlResponse extends TextResponse
+{
+	public function send(Nette\Http\IRequest $httpRequest, Nette\Http\IResponse $httpResponse): void
```
Comment (f3l1x) [inline src/Response/XmlResponse.php]: Could you please import these classes?

Comment (JanGalek) [reply src/Response/XmlResponse.php]: ok ;)

**Comment by JanGalek** [comment]
@f3l1x imported ;)

**Comment by JanGalek** [comment]
@f3l1x I too

**Pull request merged**